SDCloudUserDefaults/SDCloudUserDefaults.m

```diff
@@ -15,6 +15,37 @@ @implementation SDCloudUserDefaults
 static NSString *suiteName;
 static NSUserDefaults *userDefaults;
 
++ (void) setiCloudEnabled:(BOOL)iCloudEnabled {
+    [[NSUserDefaults standardUserDefaults] setBool:iCloudEnabled forKey:ICLOUD_DATA_ENABLED_KEY];
+    [[NSUserDefaults standardUserDefaults] synchronize];
+    
+    if (iCloudEnabled == YES) {
+        [SDCloudUserDefaults registerForNotifications];
+    } else {
+        [SDCloudUserDefaults removeNotifications];
+    }
+}
+
++ (BOOL) isiCloudEnabled {
+    return [[NSUserDefaults standardUserDefaults] boolForKey:ICLOUD_DATA_ENABLED_KEY];
+}
+
++ (void) initialize {
+    [super initialize];
```

Comment:
Is there a way that this can be implemented that doesn't require changing the the API for existing users?